CHANGELOG.md

```diff
@@ -77,10 +81,12 @@ Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/
 - BUG-000136723: Fixed an issue where MapImageLayer popup queries didn't contain time parameters.
 - BUG-000149598: Fixed an issue where popup was not returning all attributes for ImageryLayer's pixel values that equal 0.
 - BUG-000150976: Fixed an issue where the subtype coded value was displaying in popups for MapImageLayers rather than the subtype description.
+- Esri Community Idea - 940733: Attachment content file names are automatically hidden when the displayType is preview in Popups.
```

Comment:
`ArcGIS Ideas`?

Reply:
Whoops! Fixed 😄

Reply:
oh I was just going to suggest keeping it as "Esri Community - 940733" (removing the ideas)

Reply:
@bsvensson @annelfitz WDYT about keeping the ArcGIS Ideas as just Esri Community on release notes now that we are going to try to monitor these better and keep up with them? ArcGIS Ideas live on Esri Community so it would make sense to just leave it as Community and drop the Ideas.

Reply:
sounds great to me!! 👍